chia/rpc/wallet_rpc_api.py

```diff
@@ -694,20 +693,23 @@ async def push_transactions(
 
                 interface.side_effects.transactions.extend(inner_action_scope.side_effects.transactions)
 
-        return {}
+        return PushTransactionsResponse([], [])  # tx_endpoint takes care of this
```

Comment:
seems fishy to need this if it's pointless?

Reply:
Agreed, there's a tension between the two wrappers `@marshal` and `@tx_endpoint`.  `@tx_endpoint` is responsible for filling in the common `transactions` and `unsigned_transactions` arguments, but it does this at its own layer where it thinks the response is a dict (because many RPCs don't yet use `@marshal` so it is a dict there) and therefore we have to return a silly object here in order to fit the `@marshal` paradigm which requires addressing these arguments.  On the client side `PushTransactionsResponse` will have this information filled in and we could in theory fill it in in the endpoint here but it would be pointless as `@tx_endpoint` is just going to override it.

TLDR: While we sit between paradigms, this is some unfortunately necessary silliness.  It's consistent with other places in the RPC and therefore can be easily targeted in a future refactor.